packages/modal/README.md

```diff
@@ -78,16 +78,15 @@ function ExampleComponent() {
-| `initialFocus`         | `string`                          | A selector string for the element to move focus to when the modal is opened. The first focusable element in the modal will receive focus by default. |              |
-| `darkMode`             | `boolean`                         | Determines if the component will appear in dark mode.                                                                                                | `false`      |
-| `closeIconColor`       | `'default'`, `'dark'`, `'light'`  | Determines the color of the close icon. Currently will only work if `darkMode` is set to false.                                                      | `default`    |
+| Prop               | Type                              | Description                                                                                                                                          | Default      |
+| ------------------ | --------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------- | ------------ |
+| `open`             | `boolean`                         | Determines open state of `Modal` component                                                                                                           | `false`      |
+| `setOpen`          | `function`                        | Callback to set open state of Modal component. `setOpen` accepts a boolean value, which will determine the open state of `Modal` component.          | `() => {}`   |
+| `size`             | `'small'`, `'default'`, `'large'` | Determines `Modal` size.                                                                                                                             | `'default'`  |
+| `shouldClose`      | `function`                        | Callback to determine whether or not Modal should close when user tries to close it.                                                                 | `() => true` |
+| `children`         | `node`                            | Children that will be rendered inside `<Modal />` component.                                                                                         |              |
+| `className`        | `string`                          | Style to be applied to the container's root node.                                                                                                    |              |
+| `contentClassName` | `string`                          | Style to be applied to the content div.                                                                                                              |              |
+| `initialFocus`     | `string`                          | A selector string for the element to move focus to when the modal is opened. The first focusable element in the modal will receive focus by default. |              |
```

Comment:
Not sure why the diff here after prettier is so big, are these `README`s auto generated? I can remove this change if so. Let me know if it would help to add a changelog entry for this change.